src/main/scala/io/smartdatalake/util/hdfs/Partition.scala

```diff
@@ -54,6 +54,14 @@ private[smartdatalake] case class PartitionValues(elements: Map[String, Any]) {
 private[smartdatalake] object PartitionValues {
   val singleColFormat = "<partitionColName>=<partitionValue>[,<partitionValue>,...]"
   val multiColFormat = "<partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>[;(<partitionColName1>=<partitionValue>,<partitionColName2>=<partitionValue>;...]"
+
+  /**
+   * Defines an Ordering for sorting PartitionValues.
+   * Sorting a list of partition values is only possible, if the partition columns to be considered are defined.
+   * As PartitionValues is a generic structure, the type of a value needs to be inferred for comparision.
+   * @param partitions partition columns to use for sorting
+   * @return Ordering to be used e.g. with Seq.sort|sortBy
+   */
   def getOrdering(partitions: Seq[String]): Ordering[PartitionValues] = new Ordering[PartitionValues] {
     def compare(pv1: PartitionValues, pv2: PartitionValues): Int = {
       partitions.toStream.map{
```

Comment:
Why converting to stream ?
```suggestion
      partitions.map{
```
Would a sequence not be lazy enough?